src/components/WwwFrame/Header/KvAtbModalContainer.vue

```diff
@@ -116,6 +116,9 @@ const fetchBasketData = async () => {
 		},
 	}).then(({ data }) => {
 		basketData.value = data?.shop?.basket?.items?.values ?? [];
+		basketTotal.value = basketData.value.reduce((total, item) => {
```

Comment:
This looks good for mount, but does this get ran each time the modal is opened? We'd need the updated basket total each time.

Reply:
For sure, we already have that here:
https://github.com/kiva/ui/blob/58dd2e27aba159216cbb15c633301421b7646cf2/src/components/WwwFrame/Header/KvAtbModalContainer.vue#L211

Reply:
Ah yup, I missed the `addedLoan` part of the watcher, thanks!